doc/icache.rst

```diff
@@ -240,6 +240,9 @@ On other cycles, the address counter will be incremented every time an instructi
 This increment depends on the instruction data (visible at ``rdata_o``): if the bottom bits of the instruction are not ``2'b11``, the instruction is considered to be compressed and the address will be incremented by 2.
 If the bottom bits of the instruction are ``2'b11`` then the instruction is considered to be uncompressed and the address will be incremented by 4.
 Since the contents of ``rdata_o`` are not specified if an instruction fetch has caused an error, the core must signal a branch before accepting another instruction after it sees ``err_o``.
+The address in error is that signalled on ``addr_o``.
+Because a single instruction can span two 32bit memory addresses, an extra signal (``err_plus2_o``) indicates when the error is caused by the second half of an unaligned uncompressed instruction.
```

Comment:
Can compressed instructions ever signal `err_plus2_o`? Or is that squashed? Also, can `err_plus2_o` be true without `err_o`? (Is "instruction bad" equal to `err_o | err_plus2_o` or is it `err_o`, and I can figure out the offending address by looking at `err_plus2_o`?)

Reply:
Will clarify in the doc